lb_content_resolver/formats/flac.py

```diff
@@ -15,24 +16,24 @@ def get(tags, tag, default):
     return t[0]
 
 
-def read(file, mtime):
-    mdata = { 'mtime' : mtime }
+def read(file, mtime, unknown_string):
+    mdata = { "mtime" : mtime }
```

Comment:
The IDE also recommended common usage of either `'` or `"` so I went with the latter.

Reply:
:+1: